ddtrace/encoding.py

```diff
@@ -58,7 +59,35 @@ def join_encoded(objs):
 
 
 class JSONEncoder(_EncoderBase):
-    content_type = 'application/json'
+    content_type = "application/json"
+
+    @staticmethod
+    def encode(obj):
+        return json.dumps(obj)
+
+    @staticmethod
+    def decode(data):
+        return json.loads(data)
+
+    @staticmethod
+    def join_encoded(objs):
+        """Join a list of encoded objects together as a json array"""
+        return "[" + ",".join(objs) + "]"
+
+
+class JSONEncoderV2(JSONEncoder):
+    """
+    JSONEncoderV2 encodes traces to the new intake API format.
+    """
+
+    content_type = "application/json"
+
+    def encode_traces(self, traces):
+        normalized_traces = [[JSONEncoderV2._convert_span(span) for span in trace] for trace in traces]
+        return self.encode({"traces": normalized_traces})
+
+    def encode_trace(self, trace):
+        return self.encode([JSONEncoderV2._convert_span(span) for span in trace])
 
     @staticmethod
     def encode(obj):
```

Comment:
you can omit these since they inherit from the base class

Reply:
`encode`/`decode`

Reply:
Good point, I originally had some logic in here, but removed it. Will clean this up.